R/helpers.R

```diff
@@ -13,6 +13,46 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+.geocode <- function(name, country, method = "osm") {
+  if (missing(name) || !is.character(name)) {
+    stop("Please provide a valid location name as a string.")
```

Comment:
`stopifnot("Please provide a location name as a string" = is.character(name))`
Since name cannot be empty

Reply:
cleaner indeed!